pkg/scheduler/scheduler.go

```diff
@@ -115,7 +119,10 @@ func (s *scheduler) Filter(args *schedulerapiv1.ExtenderArgs) (*schedulerapiv1.E
 		glog.Infof("entering predicate: %s, nodes: %v", predicate.Name(), predicates.GetNodeNames(kubeNodes))
 		kubeNodes, err = predicate.Filter(instanceName, pod, kubeNodes)
 		if err != nil {
-			return nil, err
+			s.recorder.Event(pod, apiv1.EventTypeWarning, predicate.Name(), err.Error())
+			return &schedulerapiv1.ExtenderFilterResult{
```

Comment:
When an error occurs, we need to determine whether `kubeNodes` is empty. If it is empty, it should returns an error normally. If it is not empty, it should continue to schedule.

Reply:
fixed @onlymellb PTAL